### PR TITLE
Replace the usage of the "cockroach" driver in various tests with pgwire.

### PR DESCRIPTION
**Pull request by petermattis**
[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/4097)

### CODE REVIEW

**Comment by tamird** [comment]
Reviewed 5 of 5 files at r1.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[sql/descriptor_mutation_test.go, line 75 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/4097#-K9Z9t05K7oD9G4bBKvF:-K9Z9t05K7oD9G4bBKvG:977605559) ([raw file](https://github.com/cockroachdb/cockroach/blob/23df4c7983637b242242b9185eee209064de74fe/sql/descriptor_mutation_test.go#L75)):*
why was this needed? just wondering.

---

*[sql/main_test.go, line 126 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/4097#-K9Z99qj7C1_UyuWZZVc:-K9Z99qj7C1_UyuWZZVd:1235621471) ([raw file](https://github.com/cockroachdb/cockroach/blob/23df4c7983637b242242b9185eee209064de74fe/sql/main_test.go#L126)):*
is this ever nil?

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/4097)*

**Comment by petermattis** [comment]
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[sql/descriptor_mutation_test.go, line 75 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/4097#-K9Z9t05K7oD9G4bBKvF:-K9ZoaGNrZHDrXG30Tl8:416764022) ([raw file](https://github.com/cockroachdb/cockroach/blob/23df4c7983637b242242b9185eee209064de74fe/sql/descriptor_mutation_test.go#L75)):*
Pgwire is returning a `[]byte` instead of a `string` for some columns. `fmt.Sprintf()` (or `fmt.Sprintf("%v")` formats a `[]byte` as `[integer byte vals]`. 

---

*[sql/main_test.go, line 126 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/4097#-K9Z99qj7C1_UyuWZZVc:-K9ZoqhuJZWzmPEdPAaz:1417780758) ([raw file](https://github.com/cockroachdb/cockroach/blob/23df4c7983637b242242b9185eee209064de74fe/sql/main_test.go#L126)):*
Yeah, it's nil in tests which call `setupTestServer` directly, such as the logic tests and pgwire tests.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/4097)*

**Comment by tamird** [comment]
LGTM

---

Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/4097#-:-K9_03n1WZ40zcztbA4o:676834233)*

File: sql/scan.trace [hunk 1]
```diff
@@ -0,0 +1,12 @@
+statement ok
+CREATE TABLE t (k INT PRIMARY KEY)
+
+statement ok
+INSERT INTO t VALUES (1)
+
+traceon trace.out
```
Comment (bdarnell) [inline sql/scan.trace]: Did you mean to check this in? (maybe we should add a grep in `make check` for any test file with `traceon`)

Comment (petermattis) [inline sql/scan.trace]: Oops. No. How did that sneak in there? I don't think this needs another `make check`. I'll remove.

**Pull request merged**